src/ProjectTemplates/Web.ProjectTemplates/content/EmptyBlazorServerWeb-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,205 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "Blazor",
+    "Empty"
+  ],
+  "name": "Blazor Server App Empty",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty project template for creating a Blazor server app that runs server-side inside an ASP.NET Core app and handles user interactions over a SignalR connection. This template does not have any content in it.",
+  "groupIdentity": "Microsoft.Web.Blazor.Server.Empty",
+  "precedence": "9000",
+  "identity": "Microsoft.Web.Blazor.Server.Empty.CSharp.7.0",
+  "shortName": "blazorserver-empty",
+  "thirdPartyNotices": "https://aka.ms/aspnetcore/7.0-third-party-notices",
+  "tags": [
```

Comment:
Sorry for the confusion, I meant that this section should be in ide.host.json. for clarity, keep this in the template.json

```
  "tags": {
    "language": "C#",
    "type": "project"
  },
```

This should go in the ide.host.json:
```
  "tags": [
    {
      "type": "projectType",
      "add": [ "Cloud", "Web" ],
      "remove": [ "*" ]
    },
    {
      "type": "platform",
      "add": [ "Linux", "macOS", "Windows" ]
    }
  ],
```